gensim/models/word2vec.py

```diff
@@ -606,7 +606,7 @@ def load_word2vec_format(cls, fname, fvocab=None, binary=False, norm_only=True):
                     result.syn0[line_no] = fromstring(fin.read(binary_len), dtype=REAL)
             else:
                 for line_no, line in enumerate(fin):
-                    parts = utils.to_unicode(line).split()
+                    parts = utils.to_unicode(line.rstrip()).split(" ")
```

Comment:
Without rstrip we have one extra token in the end. The error was 
"invalid vector on line %s (is this really the text format?)"
and the line had a word, which seemingly had a space in it, but nevertheless produced by word2vec C code.